src/db.rs

```diff
@@ -106,10 +107,11 @@ struct DbInner {
 	// Overlay of most recent values int the commit queue. ColumnId -> (Key -> (RecordId, Value)).
 	commit_overlay: RwLock<Vec<HashMap<Key, (u64, Option<Value>), IdentityBuildHasher>>>,
 	log_cv: Condvar,
-	log_queue_bytes: Mutex<u64>,
+	log_queue_bytes: Mutex<i64>, // This may undeflow occasionally, but is bound for 0 eventually
```

Comment:
Saturating sub would be incorrect. It is possible for the log size to go negative, because the log file might be enacted before it's size is even added to the queue. Bu that's fine, because once it is added, the total size will be zero.

Reply:
:+1: (I should have remove this comment it was a note for myself, but good to have your comment)